server/feed-router.js

```diff
@@ -0,0 +1,59 @@
+const { Router } = require('express')
+const request = require('request')
+const { Parser } = require('xml2js')
+
+const TEST_URL = 'https://www.wired.com/feed/category/science/latest/rss'
+
+module.exports = function feedRouter() {
+  const router = new Router()
+  router.get('/', async (req, res) => {
+    try {
+      const feed = await parseXML(TEST_URL)
+      res.send(feed)
+    }
+    catch (err) {
+      console.error(err)
```

Comment:
Don't forget to respond to the client, even if there's an error. Otherwise they'll just wait until they time out.